packages/babel-plugin-remove-graphql-queries/src/index.js

```diff
@@ -337,42 +358,50 @@ export default function({ types: t }) {
           },
         })
 
+        function followVariableDeclarations(binding) {
+          const node = binding.path ? binding.path.node : undefined
```

Comment:
For future reference (not blocking on this right now) we can do `const node = binding.path?.node` here now :D

Reply:
Ah ok! Pretty nice indeed!